packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -582,6 +607,12 @@ function FilteredTelemetryView(props: FilteredTelemetryViewProps): React.ReactEl
 							onClick={(): void => {
 								setIndex(Number(rowId));
 								setSelectedEvent(item);
+								usageLogger?.sendTelemetryEvent({
+									eventName: "TelemetryEventClicked",
+									details: {
+										telemetryEventName: item.eventName,
+									},
```

Comment:
On reflection, this could actually also be generated by our users/consumers, since in Fluid we pass the logger to the container runtime and the Container and/or component authors can write their own telemetry to it. Let's remove the whole `details` from this one.